resources/help/regex.md

```diff
@@ -0,0 +1,61 @@
+# Regular expressions
+
+Blightmuds lua script offers access to powerful regular expressions. These can
+be created and used to match and replace content in strings.
+
+##
+
+***regex.new(pattern)***
+Creates a new regular expression.
+
+##
+
+***regex:test(string)***
+Checks if a string matches the regular expression.
+Returns `true` or `false`.
+
+```lua
+local re = regex.new("^This is a \\w+ line$")
+assert(re.test("This is a good line"))
```

Comment:
These should be `re:test()` I believe, not `re.test()`.

Reply:
Good spot. I'll have to patch that :smile: